modules/Header/ui/CategoriesNav/CategoriesNavDesktop.tsx

```diff
@@ -34,6 +34,19 @@ export function CategoriesNavDesktop({
         (translatedCategory) => !getCategory(translatedCategory)?.is_featured,
     );
 
+    const [currentWidth, setCurrentWidth] = useState(0);
+    useEffect(() => {
+        setCurrentWidth(document.body.offsetWidth);
+        const handleResize = () => {
+            setCurrentWidth(document.body.offsetWidth);
+        };
```

Comment:
We do prefer _`function`_ syntax when it comes to declaring functions.

```suggestion
        function handleResize() {
            setCurrentWidth(document.body.offsetWidth);
        }
```